gateway/ln-gateway/src/db.rs

```diff
@@ -23,10 +24,10 @@ pub struct FederationIdKeyPrefix;
 
 #[derive(Debug, Clone, Eq, PartialEq, Encodable, Decodable)]
 pub struct FederationConfig {
+    pub invite_code: InviteCode,
```

Comment:
This will be a breaking change for anyone running a gateway? I assume we're ok with that.

Reply:
Yeah. Okay with breaking change. But now that we have releases, should we start tracking in changelog for the next release?

Reply:
Maybe @justinmoon has thoughts on how to track breaking changes for the next release, otherwise I'm good to merge this

Reply:
dev call: we can add an identifying tag to PRs  to track breaking changes